evm_loader/program/src/entrypoint.rs

```diff
@@ -744,6 +744,34 @@ fn process_instruction<'a>(
 
             do_write(holder_info, offset, bytes)
         },
+        EvmInstruction::UpdateValidsTable => {
```

Comment:
We need a possibility to send this transaction to evm_loader. Maybe add a command `update-valids-table` in `neon-cli`?

Reply:
Done